scripts/get_version.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Simple script to get the gcloud version."""
+
+from pkg_resources import get_distribution
+print get_distribution('google-cloud').version
```

Comment:
This isn't Python 3 friendly, use

```python
print(get_distribution('google-cloud').version)
```

PS I realize I am the original author of this.